pingdom-checks/community-care.ping

```diff
@@ -3,6 +3,7 @@
 COMMUNITY_CARE_DEV_API_KEY=$(get-secret "/dvp/qa/community-care/api-key")
 COMMUNITY_CARE_SLACK_ID=$SLACK_VASDVP_MONITORING
 COMMUNITY_CARE_BACKEND_SLACK_ID=$SLACK_LIGHTHOUSE_ALERTS
+COMMUNITY_CARE_DEDICATED_ALARMS_CHANNEL_SLACK_ID=$SLACK_API_ALARMS_COMMUNITY_CARE
```

Comment:
For some of the other alerts, there is a pattern of chaining IDs together like this:
https://github.com/department-of-veterans-affairs/lighthouse-external-monitoring/blob/master/pingdom-checks/data-query.ping#L3

Rather than include `COMMUNITY_CARE_DEDICATED_ALARMS_CHANNEL_SLACK_ID` separately on each check, I think this would be simpler if `COMMUNITY_CARE_SLACK_ID` and `COMMUNITY_CARE_BACKEND_SLACK_ID` were updated to include `SLACK_API_ALARMS_COMMUNITY_CARE`